elixir/lib/extension.ex

```diff
@@ -0,0 +1,23 @@
+defmodule SqliteVss.Extension do
+  @moduledoc """
+  Extension module for adding the sqlite-vss extension to an Ecto Repo.
+
+  In order to use this add the following to the `config.exs` file.
+
+  ```
+  config :my_app, MyApp.Repo,
+    after_connect: {SqliteVss.Extension, :enable, []}
+  ```
+  """
+
+  def enable(_conn) do
```

Comment:
Can you rename this to `load(_conn)`, so it matches the other language bindings?

`SqliteVss.load(conn)` seems like a solid pattern